ex03.ex

```diff
@@ -55,7 +55,12 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([]), do: []
+  def odd_even([ h | t ]) do case Integer.is_even(h) do
```

Comment:
## -2

I'd hoped to see this without the `case`

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~